docs/portal/registry-users/managing-registry-users.md

```diff
@@ -2,4 +2,17 @@
 
 A registry user is an entity that is created and given certain permissions to access and manage repositories and tags in the registry. Usually, a registry user uses the docker CLI client to access an image registry.
 
-You manage registry users from the Registry Users page.
\ No newline at end of file
+##Registry Users Page
```

Comment:
A blankspace is missing after ##. As a result, the heading is not rendered.

Reply:
As noted above, this has been fixed. This is one example, where the heading looked fine in the VB WYSIWYG view.